dvc/command/pkg.py

```diff
@@ -54,36 +119,89 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     pkg_install_parser.add_argument(
-        "address",
-        nargs="?",
-        default="",
-        help="Package address: git://<url> or https://github.com/...",
+        "targets", nargs="*", default=[None], help="Package name."
```

Comment:
`dvc pkg install` seems to also accept URLs as `targets`, and the pkg name is extracted from there. Should the `help=` here (and other commands) be `Package name or address` ?

Reply:
p.s. but it only accepts HTTP URLs, not `git@...` addresses like `dvc pkg add`.

Reply:
> p.s. but it only accepts HTTP URLs, not git@github... addresses like dvc pkg add. Is this correct?

Fixed. Now it accepts everything that `add` accepts.